### DISCUSSION TITLE
Support cloudflare workers

### DISCUSSION THREAD

**Discussion by donferi** [sensedeep/dynamodb-onetable#200] [Ideas]
With the increasing popularity of Cloudflare Workers. Could it be possible to support their runtime?

We are dying to try it out at our org.

I looked a bit at the repo and most of the changes are around `crypto`.

**Comment by mobsense** [comment]
If it doesn't fracture the code, ....

The crypto is optional. What is the issue?

Happy to support you trying to get it to work.

**Reply by donferi** [reply]
I wrongly assumed just by importing `crypto` it was going to crash. Glad to hear it, thanks for your fast reply! We will give it a shot!
